nautobot/core/templates/utilities/object_bulk_create.html

```diff
@@ -0,0 +1,147 @@
+{% extends 'base.html' %}
```

Comment:
Having a new `utilities` templates directory in `nautobot/core/templates/` (rather than using the existing `nautobot/utilities/templates/utilities/` directory) feels wrong to me, and likely to be a source of confusion.